### PR TITLE
[ADD] connector_odbc

### PR DESCRIPTION
**Pull request by gurneyalex**
Import / synchronize data from other databases using then connector framework using ODBC.

### CODE REVIEW

**Comment by nbessi** [comment]
pyodbc must be added as dependence for travis

**Comment by clonedagain** [comment]
@lbellier we should have a look a this, to replace our odbc hacks in v8.

**Comment by gurneyalex** [comment]
green at last :sweat_smile:

File: connector_odbc/view/backend_view.xml [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+                </group>
+              </page>
+            </notebook>
+          </sheet>
+        </form>
+      </field>
+    </record>
+
+    <record id="master_data_connector_backend_list" model="ir.ui.view">
+      <field name="name">master data connector backend form</field>
```
Comment (nbessi) [inline connector_odbc/view/backend_view.xml:76]: backend list instead of form

File: connector_odbc/view/backend_view.xml [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+      <field name="name">Add an odbc import</field>
+      <field name="res_model">connector.odbc.import.configurator</field>
+      <field name="src_model">connector.odbc.data.server.backend</field>
+      <field name="view_type">form</field>
+      <field name="view_mode">form</field>
+      <field name="target">new</field>
+      <field name="help">help</field>
+    </record>
+
+    <record id="master_data_connector_backend_form" model="ir.ui.view">
```
Comment (nbessi) [inline connector_odbc/view/backend_view.xml]: I will replace the term master data with odbc

**Comment by gurneyalex** [comment]
@nbessi I fixed your last remarks.

**Comment by gurneyalex** [comment]
@guewen  could you help me? I think I've missed some changes in the connector API...

**Comment by pedrobaeza** [comment]
@gurneyalex, is this going to be continued?

**Comment by gurneyalex** [comment]
@pedrobaeza one way or another, yes. This code is in production (on v7) in one of our customers, so we will need to migrate. Maybe not to v8, though.

**Comment by pedrobaeza** [comment]
OK, then if you want it in v8, please finish the PR, squashing all Nicolas' commits and greenifying Travis, and if not, please close it.

**Comment by pedrobaeza** [comment]
Closing as no answer

**Pull request closed without merging**